app/main/views/login.py

```diff
@@ -171,3 +386,32 @@ def decode_password_reset_token(token):
         return None
 
     return decoded
+
+
+def decode_invitation_token(encoded_token):
+    try:
+        token, timestamp = decode_token(
+            encoded_token,
+            current_app.config['SHARED_EMAIL_KEY'],
```

Comment:
You beat me to it! :grin: I was just checking your other pull request and realised.